pkg/iacinventory/iacinventory.go

```diff
@@ -0,0 +1,12 @@
+package iacinventory
+
+import "github.com/soluble-ai/go-jnode"
+
+type IacInventorier interface {
+	Run() (*jnode.Node, error)
+	Stop() error
```

Comment:
What's Stop() for?  If nothing, then remove it

Reply:
Wasn't sure if I was going to need graceful handling of `^C` (deleting downloaded repos), but I haven't even implemented deleting downloaded repos...